cupy/cudnn.pyx

```diff
@@ -518,7 +518,38 @@ cpdef tuple _get_algorithm_fwd(
     return algo
 
 
-cpdef tuple _get_algorithm_bwd_filter(
+cpdef tuple _get_algorithm_fwd(
+        size_t handle, size_t x_desc, size_t filter_desc, size_t conv_desc,
+        size_t y_desc, size_t max_workspace_size, bint use_tensor_core):
+    cdef int algo
+    cdef workspace_size
+    if use_tensor_core and _cudnn_version >= 7000:
+        ret = cudnn.getConvolutionForwardAlgorithm_v7(
+            handle, x_desc, filter_desc, conv_desc, y_desc, 10)
+        i = -1
+        for j in range(len(ret)):
+            if ret[j]['memory'] <= max_workspace_size:
+                i = j
+                break
+        if i < 0:
```

Comment:
You can use `for ... else ` syntax.

Reply:
Thank you! I did not know that syntax...